### DISCUSSION TITLE
Patch 1.0.15f1 - Breaking Mod?

### DISCUSSION THREAD

**Discussion by SilverBullet126** [slyh/Cities2-TrafficLightsEnhancement#46] [General]
From loading up the game post the newest patch, I'm getting an error (multiple actually) when loading up my existing or new city :( 

![Screenshot 2023-12-01 201200](https://github.com/slyh/Cities2-Various-Mods/assets/152666726/0ccaf187-41d9-41e5-b375-df2a83bc762f)

**Comment by sysyphus** [comment]
same here waiting for update

**Reply by Thelic38** [reply]
Broken with new patch, I was also able to isolate the issue to this mod. Unfortunate as my city only functions with the help of this mod!

**Reply by harrytyp** [reply]
The mods developer is currently [working on an update](https://github.com/slyh/Cities2-Various-Mods/issues/40#issuecomment-1836919510).

**Comment by slyh** [comment]
Should be fixed in v0.2.3-alpha.
